pkg/gpu/nvidia/beta_plugin_test.go

```diff
@@ -447,6 +454,11 @@ func testNvidiaGPUManagerBetaAPIWithMig(gpuConfig GPUConfig, wantDevices map[str
 		return fmt.Errorf("failed to initilize a GPU manager")
 	}
 
+	// overriding info to mockDeviceInfo interface
+	nvmlDeviceInfo = &mockDeviceInfo{}
+	mockInfo, _ := nvmlDeviceInfo.(*mockDeviceInfo)
+	mockInfo.testDevDir = testDevDir
+
```

Comment:
Just curious, what unit test error we will get, if we didn't override testDevDir into the mockInfo?

Reply:
Hey only nvmlDeviceInfo is being overriden. mockInfo.testDevDir is a field only in mockInfo and it is being set here in the test.